src/app/workflow-editor/object-inspector/step-inspector/tabs/step-tab-info.component.ts

```diff
@@ -1,16 +1,42 @@
-import {Component, ViewEncapsulation} from "@angular/core";
-import {ComponentBase} from "../../../../components/common/component-base";
+import {Component, Input, ViewEncapsulation} from "@angular/core";
 
 @Component({
     encapsulation: ViewEncapsulation.None,
 
     selector: "ct-workflow-step-inspector-info",
     template: `
-        Info tab
+        <div>
+            <p>
+                <strong>Type:</strong>
+                <span> {{ step.run['class'] }} </span>
+            </p>
+            <p>
+                <strong>Revision:</strong>
+                <span>{{ step.run.customProps['sbg:revision'] }}</span>
+            </p>
+            <p>
+                <strong>Toolkit Version:</strong>
+                <span>{{ step.run.customProps['sbg:toolkitVersion'] }}</span>
+            </p>
+            <p>
+                <strong>Author:</strong>
+                <span> {{ step.run.customProps['sbg:createdBy'] }} </span>
+            </p>
+        
+            <div>
+                <strong>Description{{ getDescription() ? '' : ':'}}</strong>
+                <div class="form-group" [ngClass]="{'format': getDescription()}">
+                    <div [ct-markdown]="getDescription()"></div>
+                </div>
+            </div>
+        </div>        
     `
 })
-export class WorkflowStepInspectorTabInfo extends ComponentBase {
-    constructor() {
-        super();
+export class WorkflowStepInspectorTabInfo {
+    @Input()
+    public step;
+
+    private getDescription() {
+        return this.step['description'] || this.step.run['description'];
```

Comment:
`description` should be type hinted, you should access it directly through `this.step.description` and `this.step.run.description`